tests/suites/helpers.function

```diff
@@ -46,9 +46,12 @@ typedef UINT32 uint32_t;
 
 #include <string.h>
 
+#if defined(__unix__) || (defined(__APPLE__) && defined(__MACH__)) || defined(__MINGW32__)
+#include <strings.h>
+#endif
+
 #if defined(__unix__) || (defined(__APPLE__) && defined(__MACH__))
 #include <unistd.h>
-#include <strings.h>
 #endif
```

Comment:
If `__MINGW32__` is not defined but the rest of these definitions are defined then `strings.h` should still be included.

Reply:
About `strings.h` specifically, it's only needed for one call to `strcasecmp` in `test_suite_mpi.function`. (In past versions that are now all out of support, it was used in many more places.) It would be easy to do without it, so if the use of `strings.h` or the aliasing of `strcmp` to `_stricmp` is a problem, we can just get rid of it.

Reply:
My previous comment is wrong, as `string.h` is still included when `__MINGW32__` is not defined.